front/src/styles/scss/applications/operationalStudies/_scenario.scss

```
@@ -19,14 +18,58 @@
     }
   }
 
+  .micro-macro-buttons {
+    button {
+      color: var(--grey60);
+    }
+    font-size: 0.875rem;
+    margin: 1.813rem 0 2.063rem 0;
+    padding: 0.25rem 0;
+    display: inline-flex;
+    background-color: var(--ambiantB10);
+    border-radius: 0.375rem;
+    box-shadow:
+      0px 2px 2px -1px rgba(255, 171, 88, 0.27),
+      0px 1px 2px rgba(0, 0, 0, 0.16);
+    .micro-button {
+      border-right: solid 0.5px var(--grey20);
```

I'm not sure the 0.5px is working, it looks like larger than the mockup. It seems to take 1px for each border. Either you find a way to make it clean or you can also replace that by a component with this class

```suggestion
    .micro-macro-separator {
      width: 0.0625rem;
      height: 1.5rem;
      margin: -0.1875rem 0.625rem;
      background-color: var(--grey20);
    }
``

---

Done